### ISSUE TITLE
Ruby 2.2.10 > Ruby 2.2.2

### ISSUE THREAD

**Issue by osburn-sharp** [rails/rails#32455]
### Steps to reproduce

Try starting Rails 5 with Ruby 2.2.10

### Expected behavior
Should work OK

### Actual behavior
Thinks that Ruby 2.2.10 < 2.2.2 and refuses to start

### System configuration
**Rails version**: 5.1.4

**Ruby version**: 2.2.10

**Comment by rafaelfranca** [comment]
This is fixed in Rails 5.1.5

**Comment by osburn-sharp** [comment]
Thanks for letting me know. Went back to 2.2.9 to get round it on 5.1.4

**Comment by tagliala** [comment]
Hi, is this also fixed in 5.2.0.rc2? 

https://travis-ci.org/DavyJonesLocker/client_side_validations/jobs/363742643

```
Rails 5 requires Ruby 2.2.2 or newer.
    You're running
      ruby 2.2.10p489 (2018-03-28 revision 63023) [x86_64-linux]
    Please upgrade to Ruby 2.2.2 or newer to continue.
```

**Comment by riseshia** [comment]
5.2.0.rc2 "was released" before that patch, so next release will solve this problem ;)
